### PR TITLE
feat(CORSO-20): create order via the admin panel

### PR DESCRIPTION
**Pull request by DylanBrass**
JIRA: [link to jira ticket](https://champlainsaintlambert.atlassian.net/browse/CORSO-20?atlOrigin=eyJpIjoiMjRhMDg4YzQzOTU3NDg4OTljODg3MTVhMTk3OTNmOGMiLCJwIjoiaiJ9)
## Context:
The admin can now create an order from the admin dashboard, if any external orders are done.
## Changes
- Added external order dashboard
- Various life improvement changes
- Various error checking

## Before and After UI (Required for UI-impacting PRs):

No before, completely new page.

image

image

### CODE REVIEW

**Review by DenisaNicoletaH — APPROVED** [review]
The code is well done, the naming conventions are also well followed. In other words, I approve of this push

File: corso_website_springboot/src/main/java/com/corso/springboot/utils/annotations/ValidCanadianPostalCode.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Constraint(validatedBy = ValidPostalCodeRegex.class)
 public @interface ValidCanadianPostalCode {
-    String message() default "Invalid Phone Number Format.";
+    String message() default "Invalid Postal Code Format.";
 
     Class<?>[] groups() default {};
```
Comment (Mila5847) [inline corso_website_springboot/src/main/java/com/corso/springboot/utils/annotations/ValidCanadianPostalCode.java:17]: Can you explain the Class<?> groups() default{}; please

**Review by Mila5847 — APPROVED** [review]
Looks good, tested and the database tables are updated

File: corso_website_springboot/src/main/java/com/corso/springboot/utils/annotations/ValidCanadianPostalCode.java (hunk 1 above)
Comment (DylanBrass) [reply corso_website_springboot/src/main/java/com/corso/springboot/utils/annotations/ValidCanadianPostalCode.java:17]: It allows the specification of validation groups, to which this constraint belongs, in other words nothing important for our use case, but it's required for the Custom constraint to be a valid one.

**Pull request merged**